compiler-access/src/main/scala/org/alephium/ralph/lsp/access/compiler/RalphParserExtension.scala

```diff
@@ -1,9 +1,11 @@
 package org.alephium.ralph.lsp.access.compiler
 
 import fastparse._
+import org.alephium.ralph.Ast
+import org.alephium.ralph.SourceIndex
+import org.alephium.ralph.StatefulParser
```

Comment:
are you sure you want this change? why not keeping the grouping in `{ ... }`?

Reply:
You are right. I didn't realise it changed. Lemme revert this.

Reply:
Forced pushed to clear out the bad commit. Hope that's ok.